CONTRIBUTING.md

```diff
@@ -33,42 +33,46 @@ The following people have contributed to this code under the terms of
 the Contributor Licence Agreement and Certificate of Origin detailed
 below:
 
-* Paul Abernethy (Met Office, UK)
-* Benjamin Ayliffe (Met Office, UK)
-* Mark Baker (Met Office, UK)
-* Laurence Beard (Met Office, UK)
-* James Canvin (Bureau of Meteorology, Australia)
-* Neil Crosswaite (Met Office, UK)
-* Gavin Evans (Met Office, UK)
-* Zhiliang Fan (Bureau of Meteorology, Australia)
-* Ben Fitzpatrick (Met Office, UK)
-* Martina Friedrich (Met Office, UK, pre-GitHub)
-* Tom Gale (Bureau of Meteorology, Australia)
-* Aaron Hopkinson (Met Office, UK)
-* Kathryn Howard (Met Office, UK)
-* Tim Hume (Bureau of Meteorology, Australia)
-* Katharine Hurst (Met Office, UK)
-* Simon Jackson (Met Office, UK)
-* Caroline Jones (Met Office, UK)
-* Bruno P. Kinoshita (NIWA, NZ)
-* Lucy Liu (Bureau of Meteorology, Australia)
-* Daniel Mentiplay (Bureau of Meteorology, Australia)
-* Stephen Moseley (Met Office, UK)
-* Meabh NicGuidhir (Met Office, UK)
-* Benjamin Owen (Bureau of Meteorology, Australia)
-* Carwyn Pelley (Met Office, UK)
-* Tim Pillinger (Met Office, UK)
-* Fiona Rust (Met Office, UK)
-* Chris Sampson (Met Office, UK)
-* Caroline Sandford (Met Office, UK)
-* Victoria Smart (Met Office, UK)
-* Eleanor Smith (Met Office, UK)
-* Marcus Spelman (Met Office, UK)
-* Belinda Trotta (Bureau of Meteorology, Australia)
-* Tomasz Trzeciak (Met Office, UK)
-* Mark Worsfold (Met Office, UK)
-* Shubhendra Singh Chauhan (DeepSource, India)
-* Ying Zhao (Bureau of Meteorology, Australia)
+<!-- start-shortlog -->
+ - Paul Abernethy (Met Office, UK)
+ - Benjamin Ayliffe (Met Office, UK)
+ - Mark Baker (Met Office, UK)
+ - Laurence Beard (Met Office, UK)
+ - Anna Booton (Met Office, UK)
+ - James Canvin (Bureau of Meteorology, Australia)
+ - Neil Crosswaite (Met Office, UK)
+ - Shafiat Dewan (Met Office, UK)
+ - Gavin Evans (Met Office, UK)
+ - Zhiliang Fan (Bureau of Meteorology, Australia)
+ - Ben Fitzpatrick (Met Office, UK)
+ - Tom Gale (Bureau of Meteorology, Australia)
+ - Aaron Hopkinson (Met Office, UK)
+ - Kathryn Howard (Met Office, UK)
+ - Tim Hume (Bureau of Meteorology, Australia)
+ - Katharine Hurst (Met Office, UK)
+ - Simon Jackson (Met Office, UK)
+ - Caroline Jones (Met Office, UK)
+ - Bruno P. Kinoshita (NIWA, NZ)
+ - Lucy Liu (Bureau of Meteorology, Australia)
+ - Daniel Mentiplay (Bureau of Meteorology, Australia)
+ - Stephen Moseley (Met Office, UK)
+ - Meabh NicGuidhir (Met Office, UK)
+ - Benjamin Owen (Bureau of Meteorology, Australia)
+ - Carwyn Pelley (Met Office, UK)
+ - Tim Pillinger (Met Office, UK)
+ - Fiona Rust (Met Office, UK)
+ - Chris Sampson (Met Office, UK)
+ - Caroline Sandford (Met Office, UK)
+ - Victoria Smart (Met Office, UK)
+ - Eleanor Smith (Met Office, UK)
+ - Marcus Spelman (Met Office, UK)
+ - Belinda Trotta (Bureau of Meteorology, Australia)
+ - Tomasz Trzeciak (Met Office, UK)
+ - Mark Worsfold (Met Office, UK)
+ - Shubhendra Singh Chauhan (DeepSource, India)
```

Comment:
Is the alphabetical ordering a little off here?

Reply:
It is - not me! I will rearrange it